cmd/syncer/cmd/syncer.go

```diff
@@ -53,7 +53,11 @@ func NewSyncerCommand() *cobra.Command {
 			}
 
 			ctx := genericapiserver.SetupSignalContext()
-			if err := Run(options, ctx); err != nil {
+
+			logger := klog.Background()
+			ctx = klog.NewContext(ctx, logger)
```

Comment:
We already have a `ctx` from `SetupSignalContext` above, is there a reason we can't just call `klog.FromContext` using that object?

Reply:
You're right, `klog.FromContext()` will return `klog.Background()` if there isn't a logger in the context. Will adjust.